Patch by Lars Kanis
https://github.com/ruby/ruby/pull/1591
https://github.com/ruby/ruby/commit/c483ae6572f4234ffbe6f3da98e0d91492b09a2a
MSP-Greg note - should only be needed for svn < 58423  2017-04-20 & vers >= 2.5
diff --git a/include/ruby/win32.h b/include/ruby/win32.h
index ea96373..e8cb5e3 100644
--- a/include/ruby/win32.h
+++ b/include/ruby/win32.h
@@ -783,7 +783,7 @@ RUBY_SYMBOL_EXPORT_END
 static inline double
 rb_w32_pow(double x, double y)
 {
-    return powl(x, y);
+    return (double)powl(x, y);
 }
 #elif defined(__MINGW64_VERSION_MAJOR)
 double rb_w32_pow(double x, double y);
